src/doc/rustc-dev-guide/src/building/bootstrapping/what-bootstrapping-does.md

```diff
@@ -84,26 +87,25 @@ The rustc source code is then compiled with the `stage0` compiler to produce the
 
 ### Stage 2: the truly current compiler
 
-We then rebuild our `stage1` compiler with itself to produce the `stage2`
+We then rebuild our `stage1` compiler with in-tree std to produce the `stage2`
```

Comment:
nit:
```suggestion
We then rebuild the compiler using `stage1` compiler with in-tree std to produce the `stage2`
```

Rebuilding `stage1` doesn't make sense (it would imply that we are producing stage1!) and I think it's useful to note that we are using stage1 compiler here.